platforms/hyperledger-fabric/configuration/roles/create/configtx/templates/configtxOrderer_default.tpl

```diff
@@ -21,19 +21,25 @@ Orderer: &OrdererDefaults
 {% if consensus.name == 'kafka' %}
   Kafka:
     Brokers:
+{% for org in network.organizations %}
+{% if org.services.orderers is defined and org.services.orderers|length > 0 %}
 {% for i in range(consensus.replicas) %}
-      - {{ consensus.name }}-{{ i }}.{{ consensus.type }}.{{ component_ns }}.svc.cluster.local:{{ consensus.grpc.port }}
+      - {{ consensus.name }}-{{ i }}.{{ consensus.type }}.{{ org.name |lower }}-net.svc.cluster.local:{{ consensus.grpc.port }}
```

Comment:
Q: Is there an assumption here that all orderers will be in the same cluster?

Reply:
Yes, for Kafka this was the assumption we went with, just changing the invalid variable here